lib/Arch/X86/Arch.cpp

```diff
@@ -788,6 +788,66 @@ static void DecodeOperand(Instruction &inst, const xed_decoded_inst_t *xedd,
   }
 }
 
+static uint64_t BytesToCallBits(const uint8_t *bytes) {
+  // The pattern we're looking for is 5 bytes long.
+  uint64_t bits = 0;
+  bits = (bits << 8) | static_cast<uint32_t>(bytes[0]);
```

Comment:
For loop?

Reply:
It seems like a `memcmp` again the bytes in a static const array could also work.

Reply:
[de4ec45](https://github.com/lifting-bits/remill/pull/610/commits/de4ec451c95237019aa5a1c7c9f990f3ae492afb)